app/src/main/java/com/rockthevote/grommet/ui/MainActivityViewModel.kt

```diff
@@ -105,14 +107,23 @@ class MainActivityViewModel(
      */
     fun loadSessionStatus() {
         viewModelScope.launch(dispatchers.io) {
-            val retainedStatus  = sessionDao.getCurrentSession()?.sessionStatus
+            val rawState = sharedPreferences.getString(KEY_SESSION_STATUS, null)
 
-            val status = retainedStatus ?: SessionStatus.PARTNER_UPDATE
+            val status = SessionStatus.fromString(rawState) ?: SessionStatus.PARTNER_UPDATE
```

Comment:
Should this be PARTNER_UPDATE or something else?

Reply:
partner update is the first step, so this is correct